packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -213,8 +212,7 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
   itemRefs = [] as React.RefObject<HTMLElement>[]
   paddleNextRef = React.createRef<HTMLElement>()
   paddlePreviousRef = React.createRef<HTMLElement>()
-
-  focusItemAtIndex = debounce((index: number) => {
+  focusItemAtIndex = _.debounce((index: number) => {
     this.itemRefs[index].current.focus()
```

Comment:
If you're debouncing, you'll need to check `current` for null. There is a slight race condition where the debounced callback can occur asynchronously after unmounting.